### PR TITLE
GTW-7651: Split 3ds request into 3rd party and standalone

### PR DESCRIPTION
**Pull request by sam-bonheure-cko**
# Description of changes in PR

[//]: # 'Please add your description here'
Add new fields for payments using standalone authentication and add OneOf to differentiate 3ds information

## Checklist

- [x] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [x] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

File: nas_spec/components/schemas/Payments/3dsRequestThirdParty.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+type: object
+title: Third Party 3DS authentication
+description: Required Third Party 3DS authentication information
```
Comment (jason-dantzer-cko) [inline nas_spec/components/schemas/Payments/3dsRequestThirdParty.yaml:3]: "Required information to process a payment that has been 3DS authenticated [using a third-party provider](https://www.checkout.com/docs/payments/regulation-support/3d-secure-payments-with-a-third-party-provider#The_request)

**Review by jason-dantzer-cko — CHANGES_REQUESTED** [review]

File: nas_spec/components/schemas/Payments/3dsRequestThirdParty.yaml (hunk 1 above)
Comment (sam-bonheure-cko) [reply nas_spec/components/schemas/Payments/3dsRequestThirdParty.yaml]: [Update descriptions](https://github.com/checkout/checkout-api-reference/pull/661/commits/4b3afe301b8b0229a549bb2e8fc9a5074f99a308)

File: nas_spec/components/schemas/Payments/3dsRequestStandalone.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+type: object
+title: Standalone 3DS authentication
+description: Required Standalone 3DS authentication information
```
[the remark this answers is not in the document]
Comment (sam-bonheure-cko) [reply nas_spec/components/schemas/Payments/3dsRequestStandalone.yaml]: [Update descriptions](https://github.com/checkout/checkout-api-reference/pull/661/commits/4b3afe301b8b0229a549bb2e8fc9a5074f99a308)

**Review by Joshua-Bedeau-cko — DISMISSED** [review]

**Comment by sebastian-garcia-cardona-cko** [comment]
@sam-bonheure-cko just a reminder that your branch has merge conflicts. You'll need to resolve these so reviewers can approve

**Review by jason-dantzer-cko — APPROVED** [review]

**Review by sebastian-garcia-cardona-cko — APPROVED** [review]

**Pull request merged**